python/src/ExpectationSimulationAlgorithm_doc.i.in

```diff
@@ -11,70 +11,262 @@ X : :class:`~openturns.RandomVector`
 
 Notes
 -----
-The algorithm can operate on a multivariate random vector :math:`\vect{X}`.
+This algorithm estimates :math:`\Expect{\vect{X}}` using the smallest possible sample 
+size which satisfies pre-defined stopping rules. 
+It increases the sample size :math:`n` incrementally until a stopping criterion is met. 
+
+The algorithm is based on two nested loops:
+
+- the outer loop sets the number of iterations of the algorithm; 
+  this can be configured using :meth:`setMaximumOuterSampling`,
+
+- the inner loop sets the number of function calls which can be parallelized; 
+  this can be configured using :meth:`setBlockSize`.
+
+The algorithm allows to get the best possible performance on distributed 
+supercomputers and multi-core workstations. 
+For example, if the block size is equal to 100, then the sample size 
+is successively equal to 100, 200, etc... 
```

Comment:
Isn't it contradictory with the first sentence, i.e. the claim that the algorithm estimates the smallest possible sample size which satisfies pre-defined stopping rules?

Reply:
You are right. Fixed: "This algorithm estimates E(X) based on a trade-off between performance and accuracy, using the smallest possible sample (blockwise) size which satisfies pre-defined stopping rules. "